test-runner.sh

```diff
@@ -0,0 +1,10 @@
+cd frontend
```

Comment:
given that tests weren't adequately failing using this do we still want to use it?